### DISCUSSION TITLE
why is bf facing the other way

### DISCUSSION THREAD

**Discussion by ChickenMcNug1357** [KadeArchive/Kade-Engine#1849] [Q&A]
I went to the debug and went into week 3 to add my character and when I loaded in, bf was facing the other way and idk why its doing this but idk how to fix it
![Screenshot (21)](https://user-images.githubusercontent.com/89220536/130131286-965e2258-0f68-436b-bd79-5bf7ef6fe5ce.png)

**Comment by napstaa967** [comment]
does this happen on the default mode?

if yes:
character.hx
go to the part where boyfriend's assets are
go after playAnim('idle')
invert flipx (true if its false, false if its true)

**Reply by ChickenMcNug1357** [reply]
IT WORKED, thank you i was so confused, im new to modding and i was so confused
